gguf-py/gguf/tensor_mapping.py

```diff
@@ -161,18 +161,20 @@ class TensorNameMap:
             "encoder.layer.{bid}.intermediate.dense",                 # bert
             "transformer.h.{bid}.mlp.fc_in",                          # gpt-j
             "language_model.encoder.layers.{bid}.mlp.dense_h_to_4h",  # persimmon
+            "transformer.h.{bid}.mlp.w1",                             # qwen
         ),
 
         # Feed-forward gate
         MODEL_TENSOR.FFN_GATE: (
             "model.layers.{bid}.mlp.gate_proj",  # llama-hf refact
             "layers.{bid}.feed_forward.w1",      # llama-pth
+            "transformer.h.{bid}.mlp.w2",        # qwen
```

Comment:
Is it possible that `FFN_GATE` and `FFN_UP` are swapped during the conversion?

Reply:
Hm, based on https://huggingface.co/Qwen/Qwen-7B/blob/e7a368b0774370edec29674e7c51f52fc7663f59/modeling_qwen.py#L572-L577, these are correct

There is something else going on - not sure why the quantum models break